ansible/roles/discover/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Install discovery-related packages
+  apt:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - avahi-daemon
+    - avahi-utils
+    - libnss-mdns
+    - dnsmasq
+
+- name: Configure avahi-daemon
+  template:
+    src: avahi/avahi-daemon.conf
+    dest: /etc/avahi/avahi-daemon.conf
```

Comment:
So I guess the answer is "we run it from supervisor". Then let's move its config file to `/var/lib/liquid/conf` somewhere.